modules/main-rules/pages/roll-dice-and-add-firefight-bonuses.adoc

```diff
@@ -1,6 +1,9 @@
-= FF3. Roll dice and add firefight bonuses
+= FF3. Make the combat resolution roll
 
+.Combat resolution roll
```

Comment:
Might we need to disambiguate? Perhaps we actually have 'close combat resolution roll' and 'firefight resolution roll'?
Collectively, if/where necessary, 'assault resolution roll'.

Reply:
I don't think we gain anything from having two different names. 

We could rename it to 'assault resolution roll', but I suspect that'll just mean that players use a different term than we use in the rules... :-/ I picked 'combat resolution' because that's what everyone seems to use, regardless of game system.

Reply:
For my liking, 'combat resolution' is much too close to 'close combat resolution' — as distinct from 'firefight resolution'. And we don't call it the 'Combat phase' in this edition, we call it the 'Assault phase'.

I suggest that regardless of what terminology a sample of people may use on casual basis, it pays to be precise and to disambiguate in the actual text. And it's better to be consistent I think — Assault orders, Assault phase, assault move, assault resolution roll.

So I do think that for the general case, 'assault resolution' would be better. And turning my original thought around, we'd use that one term but then qualify it where we need to disambiguate/be specific — 'assault resolution roll for close combat' and 'assault resolution roll for firefight'.

Reply:
> For my liking, 'combat resolution' is much too close to 'close combat resolution' — as distinct from 'firefight resolution'. And we don't call it the 'Combat phase' in this edition, we call it the 'Assault phase'.

It's not called the Combat phase or Combat orders in EA either, but the only term I can remember being used across multiple events is 'combat resolution' because that's the standard wargaming term. Note that the original Epic 40k rules use 'the combat' to describe both close combats and firefights.

Personally, I'm very hesitant about choosing terminology that I know is not going to be used by (in my experience) a large proportion of players, but obviously you're the person who has the final say on that.

> And turning my original thought around, we'd use that one term but then qualify it where we need to disambiguate/be specific — 'assault resolution roll for close combat' and 'assault resolution roll for firefight'.

I assume you mean when referring to it in other places, and not within the two resolution pages themselves?

Reply:
P.S. My preferred page title would have simply been 'Combat resolution', but of course that then doesn't fit with the rest of the close combat and firefight procedure page titles. :-(

Reply:
I'm pretty well decided on 'Assault resolution' I think — but I'll try to expound my rationale if only for the sake of posterity:

>Personally, I'm very hesitant about choosing terminology that I know is not going to be used by (in my experience) a large proportion of players, but obviously you're the person who has the final say on that.

I'm not especially, to be honest — people use inaccurate terminology for things all the time, but it doesn't mean that the actual source materials should give up on precision and match colloquial use.

>It's not called the Combat phase or Combat orders in EA either, but the only term I can remember being used across multiple events is 'combat resolution' because that's the standard wargaming term. Note that the original Epic 40k rules use 'the combat' to describe both close combats and firefights.

My point is more that regardless of other editions/games and colloquial terminology, in this edition/game it's called the Assault phase, and the terminology should be consistent with that. The terminology shouldn't wander off and overlap with a sub-category. Terminology isn't as robust as it could be in EA either, but in this case it doesn't present such an issue for one important reason: EA close combats and firefights basically work the same way and share **one procedure**, so it doesn't really matter what you call it — unlike E40k which has a similar but separate procedure for each, and the one can happen after the other.

>I assume you mean when referring to it in other places, and not within the two resolution pages themselves?

Mainly, yes — but when I look again to edit, it might be that the qualifiers are useful to ensure clarity. In fact, see my thoughts on the titles below.

>P.S. My preferred page title would have simply been 'Combat resolution', but of course that then doesn't fit with the rest of the close combat and firefight procedure page titles. :-(

It's good practice to start all 'tasks' in a 'procedure' with a verb. So actually, neither "Combat resolution roll" nor "Combat resolution" are great.

Thus I'm inclined towards something more like "Roll to resolve the assault". Other possibilities include "Roll for assault resolution" or "Make the assault resolution roll", but these are a bit inelegant IMHO.
Or, given that we're talking about two similar but non-identical topics inside two separate procedures, it might pay to include the qualifiers in the titles "Roll to resolve the assault (close combat)" and "Roll to resolve the assault (firefight)". That way we can still refer to "re-roll the roll to resolve the assault" and cover both cases.

Reply:
This conversation is now outdated by various changes, but I'll keep an eye open for other instances where terminology needs to change to be consistent.